src/helpers/Monitor.cpp

```diff
@@ -343,6 +357,30 @@ bool CMonitor::isMirror() {
     return pMirrorOf != nullptr;
 }
 
+bool CMonitor::matchesString(const std::string& name) {
```

Comment:
if you're adding this helper here, it'd be nice to use it in appropriate places like e.g. `CCompositor::getMonitorFromString`, otherwise they can easily fall apart later